playbooks/templates/.env.j2

```diff
@@ -293,6 +293,11 @@ ABUSE_PORTAL_URL={{ webportal_common_config.abuse_portal_url }}
 # REQUIRED: Hardcoded, not configurable
 ABUSE_SPONSOR="Abuse Scanner"
 
+# Temporary directory used by the abuse scanner
+#
+# OPTIONAL: manually defined by user
+ABUSE_TMP_DIR={{ webportal_common_config.abuse_tmp_dir | default('/tmp/abuse-scanner') }}
```

Comment:
What is this used for?

A tmp directory doesn't feel like a needed environment variable.  If it is just used during setup then we can just have an ansible variable.

Reply:
I can throw this out. I thought we usually add an optional variable for every environment variable we use from the environment so it's configurable should portal operators want to deviate from the default value.